imager/imager_profile/tests.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import unicode_literals
+from django.contrib.auth.models import User
+from django.test import TestCase
+from imager.imager_profile.models import UserProfile
+
+import factory
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+
+
+class ProfileTestCase2(TestCase):
+    """Define a test case to see if testing works."""
```

Comment:
Defining a test to see if testing works is not really a valuable test. If something breaks within Django's testing system, Django's tests for Django's testing system will catch those.

In general, you should only be writing tests for your own logic, to make sure that you've hooked the right wires up to the right things in the right ways. There's no need to test that Django itself works. My general rule of thumb when decided what tests are valuable to write is: "Assume that Django works perfectly, in just the way it's supposed to. Assume that I've screwed everything up. Write tests to make sure that I haven't."